src/Action/TrackDownloads.php

```diff
@@ -39,15 +39,14 @@ public function __invoke(Request $request, Response $response, $workshop, $versi
         try {
             $workshop = $this->workshopRepository->findByCode($workshop);
         } catch (RuntimeException $e) {
-            return new JsonResponse(
-                ['status' => 'error', 'message' => sprintf('Workshop: "%s" not found.', $workshop)]
-            );
+            return (new SlimResponse())
```

Comment:
For whatever reason `Diactoros` is not installed anymore, I suspect it was a dep of something else and was dropped when the zend libs were upgraded.